content/en/tracing/profiler/getting_started.md

```diff
@@ -256,6 +256,88 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 [6]: /tracing/visualization/#services
 [7]: /tracing/guide/setting_primary_tags_to_scope/#environment
 {{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+<div class="alert alert-warning">
+Datadog Ruby Profiler is currently in public beta. We strongly advise evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The Datadog Profiler requires MRI Ruby 2.1+ and **CPU-time profiles are only available on Linux platforms**. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][1]+ or [6.20.2][1]+.
+
+2. Add the `ddtrace` and `google-protobuf` gems to your `Gemfile` or `gems.rb` file:
+
+    ```ruby
+    gem 'ddtrace'
+    gem 'google-protobuf', '~> 3.0'
+    ```
+
+    **Note**: Profiler is available in the `ddtrace` library for versions 0.49+.
+
+2. Install the gems with `bundle install`
+
+3. You can auto-enable the profiler by setting the `DD_PROFILING_ENABLED` environment variable to `true`:
+
+    ```shell
+    DD_PROFILING_ENABLED=true \
+    DD_ENV=prod \
+    DD_SERVICE=my-web-app \
+    DD_VERSION=1.0.3
+    ```
+
+    **Note**
+
+    You can also configure the profiler via code. Rails applications you can create a `config/initializers/datadog.rb` file with Profiling enabled:
+
+    ```ruby
+    Datadog.configure do |c|
+
+      # This will enable the profiler
+      c.profiling.enabled = true
+    end
+    ```
+
+4. Add the `ddtracerb exec` command to your Ruby application start command:
+
+    ```shell
+    bundle exec ddtracerb exec ruby myapp.rb
+    ```
+
+    Rails example:
+
+    ```shell
+    bundle exec ddtracerb exec bin/rails s
+    ```
+
+    **Note**
+
+    If starting the application via `ddtracerb exec` is not an option (eg. when using the Phusion Passenger web server), you can alternatively start the profiler by adding the following to your application entry point such as `config.ru` for a web application:
+
+    ```ruby
+    require 'ddtrace/profiling/preload'
+    ```
+
+
+4. After a minute or two of starting your Ruby application, visualize your profiles in the [Datadog APM > Profiler page][2].
+
+**Note**:
```

Comment:
I don't think we need this whole note section anymore; all the content is already covered above.